addons/medical/XEH_postInit.sqf

```diff
@@ -31,6 +31,20 @@ GVAR(heartBeatSounds_Slow) = ["ACE_heartbeat_slow_1", "ACE_heartbeat_slow_2"];
 if (isServer) then {
     ["ace_placedInBodyBag", FUNC(serverRemoveBody)] call CBA_fnc_addEventHandler;
     [QGVAR(createLitterServer), FUNC(handleCreateLitter)] call CBA_fnc_addEventHandler;
+    addMissionEventHandler ["BuildingChanged", {
+        if (isNil QGVAR(allCreatedLitter)) exitWith {};
+        params ["_buildingOld", "_buildingNew", "_isRuin"];
```

Comment:
Shouldn't `_isRuin` need to be checked before cleaning up?

Reply:
Some buildings might change appearance enough to make some things float before they are ruins.